app/models/section.rb

```diff
@@ -112,7 +116,38 @@ def self.from_publishing_api(content_id:)
     section
   end
 
-private
+  def updated_manual_links?
```

Comment:
Just looking at this again... I wouldn't expect this method to update the Publishing API based on its name.